### DISCUSSION TITLE
Does anyone have any spare open muscle boards?

### DISCUSSION THREAD

**Discussion by xgpt** [turfptax/openmuscle#4] [Q&A] [answered]
I'd be very happy to send enough money to cover the cost of the board + shipping + a small handling fee for one of the boards. I'd like to start helping out but don't want to do a full board order just for the one board I would need. I've already got the WeMos boards on hand from other projects.

**Comment by turfptax** [comment, marked as answer]
I appreciate your willingness to help. I would be happy to send you some boards. There are 6 used for a complete bracelet for version 5.3.0. I am currently working on designing version 6 that should come with some decent wiring management solutions.

If you want to send a message through the site I can work on getting you some boards: https://openmuscle.org/7b3c2-contact/
